Automattic/wp-calypso
# Better support for @wordpress/element Fragment JSX

See https://github.com/WordPress/gutenberg/pull/15120

Matches Gutenberg babel preset for JSX:
https://github.com/WordPress/gutenberg/blob/af6e5ecd8b398e08127ad1b3c784ce97899de100/packages/babel-preset-default/index.js#L58-L70

Provides better `calypso-build` support for `<></>`.

#### Testing instructions

* Extensions and apps built with `calypso-build` should continue to work as before.

Provided that `<></>` syntax was basically unsupported by the previous setup, this should introduce few or no changes for consumers.

It will allow `<Fragment></Fragment>` to be replaced by `<></>`, leveraging the correct `@wordpress/element` Fragment.
